scripts/minimal-integration-test.js

```
@@ -0,0 +1,125 @@
+
+  try {
+    const client = new CloudflareSpeedTestClient();
+    let passed = 0;
+    let total = 0;
+
+    // Test 1: Connection Info
+    total++;
+    console.log('📡 Testing connection info...');
+    try {
+      const info = await client.getConnectionInfo();
+      if (info.ip && info.ip.match(/^\d+\.\d+\.\d+\.\d+$/)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)